nucleus/src/main/java/za/co/absa/subatomic/infrastructure/team/view/jpa/TeamRepository.java

```diff
@@ -2,9 +2,15 @@
 
 import org.springframework.data.jpa.repository.JpaRepository;
 
+import java.util.List;
+
 public interface TeamRepository extends JpaRepository<TeamEntity, Long> {
 
     TeamEntity findByTeamId(String teamId);
 
     TeamEntity findByName(String name);
+
+    List<TeamEntity> findByMembers_SlackDetailsScreenName(String screenName);
```

Comment:
Do these query methods work?

Given this is going to have to do a SQL join, does using just the query method work without having to use `@Query(...)` with an explicit join etc.

Reply:
Yeah they do. I have tested.

See this too:
https://docs.spring.io/spring-data/jpa/docs/current/reference/html/#repositories.query-methods.query-property-expressions

Reply:
Will test after merge.

Reply:
Does indeed work 👍 

I have now learnt something, didn't know you could handle joins like this. Would've done some `@Query(... join ...)` stuff otherwise. 😄

Reply:
IntelliJ doesn't like it though:

![screen shot 2018-02-17 at 07 23 41](https://user-images.githubusercontent.com/1175891/36338378-ff0def1e-13b5-11e8-80d5-a29dbdc50715.png)